app/assets/javascripts/analytics/init.js.erb

```diff
@@ -99,6 +99,7 @@ var analyticsInit = function() {
       'gro.gov.uk',
       'helpwithcourtfees.service.gov.uk',
       'help-with-prison-visits.service.gov.uk',
+      'coronavirus-shielding-support.service.gov.uk',
```

Comment:
Could this keep with convention and be listed alphabetically?

Reply:
I had a glitch and thought I was pasting in "jobseekers" for some reason but that was a different thing 😓